urllib3/response.py

```diff
@@ -387,48 +386,67 @@ def readinto(self, b):
             b[:len(temp)] = temp
             return len(temp)
 
-    def read_chunked(self, amt=None):
+    def _get_next_chunk(self):
+        # First, we'll figure out length of a chunk and then
+        # we'll try to read it from socket.
+        if self.chunk_left is None:
+            line = self._fp.fp.readline()
+            line = line.decode()
+            # See RFC 7230: Chunked Transfer Coding.
+            i = line.find(';')
+            if i >= 0:
+                line = line[:i]  # Strip chunk-extensions.
```

Comment:
Done in c1317ff but using a slightly different way around the fact that `line` will be `bytes`.